### PR TITLE
style: APP-393 align dollar sign in small screens

### PR DESCRIPTION
**Pull request by r41ph**
## Description

https://regennetwork.atlassian.net/browse/APP-393

### Author Checklist

_All items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues._

I have...

- [x] provided a link to the relevant issue or specification
- [x] provided instructions on how to test
- [x] reviewed "Files changed" and left comments if necessary
- [x] confirmed all CI checks have passed

### How to test

1. Go to the Buy Credits form (https://deploy-preview-2509--regen-marketplace.netlify.app/project/C01-001/buy) and with USD selected verify that the dollar sign is properly aligned on both mobile and desktop screens.

### Reviewers Checklist

_All items are required. Please add a note if the item is not applicable and please **add
your handle next to the items reviewed if you only reviewed selected items**._

I have...

- [ ] confirmed all author checklist items have been addressed
- [ ] reviewed code correctness and readability
- [ ] verified React components follow DRY principles
- [ ] reviewed documentation is accurate
- [ ] reviewed tests
- [ ] manually tested (if applicable)

### CODE REVIEW

**Comment by r41ph** [comment]
@erikalogie see testing instructions

**Comment by erikalogie** [comment]
LGTM though I think there could be a 2-3 px more space between the $ sign and the number on mobile.

**Comment by blushi** [comment]
This doesn't look like on figma where the $ is perfectly aligned with the amount on both desktop and mobile

image

image

**Comment by r41ph** [comment]
Good catch. In the Order Summary the $ is above the amount and in the currency field is next to the amount. 

>LGTM though I think there could be a 2-3 px more space between the $ sign and the number on mobile.

@erikalogie Do you want it to look like in Figma or the currency field should look like the Order Summary?

image

**Comment by erikalogie** [comment]
I think in the interest of time, how you've currently implemented it is a great improvement and we should just go with this for now and can tweak it more later!

**Comment by erikalogie** [comment]
LGTM!

File: web-marketplace/src/components/molecules/CreditsAmount/CurrencyInput.tsx [hunk 1]
```diff
@@ -77,15 +77,17 @@ export const CurrencyInput = ({
...
+        </span>
       )}
       <TextField
         {...register(CURRENCY_AMOUNT)}
         onChange={handleOnChange}
         type="number"
         className={`border border-solid border-grey-300 focus-within:border-grey-500 focus-within:border-2 ${
           paymentOption === PAYMENT_OPTIONS.CARD ? 'pl-5' : ''
-        } w-full sm:w-auto flex justify-start relative sm:h-60 rounded-sm items-center`}
+        } w-full sm:w-auto flex justify-start relative xs:h-60 rounded-sm items-center`}
```
Comment (blushi) [inline web-marketplace/src/components/molecules/CreditsAmount/CurrencyInput.tsx:90]: we don't have `xs` breakpoint so `xs:h-60` doesn't do anything

Comment (r41ph) [reply web-marketplace/src/components/molecules/CreditsAmount/CurrencyInput.tsx:90]: That was a mistake, thanks for catching it. I’ll revert it back.

**Pull request merged**